pages/paradicepalase-jovanniluna.html

```
@@ -168,14 +168,16 @@ <h1>“The Process of Jovanni Luna at Paradice Palase”</h1>
       </section>
 
+      <p><strong>Related on I.A.M.:</strong> <a href="./deancercone.html">Dean Cercone at Galerie Manque</a> &middot; <a href="./bushwick.html">Bushwick: Where Does It Rank</a> &middot; <a href="./carlitodalceggio-interview.html">Carlito Dalceggio Interview</a></p>
+
       <section
         class="module parallax parallax-2"
         data-type="background"
         data-speed="10"
       >
-      
+
         </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)